File: nowcoder/BC33_计算平均成绩/BC33.c
#include<stdio.h>
int main()
{
    int a, b, c, d, e;
    double sum, avg;
    scanf("%d%d%d%d%d", &a, &b, &c, &d, &e);
    sum = a + b + c + d + e;
    avg = sum / 5;
    printf("%.1f", avg);
}

File: nowcoder/BC33_计算平均成绩/BC33.go
package main
 
import "fmt"
 
func main() {
    var a,b,c,d,e int
    var sum,avg float64
    fmt.Scanf("%d%d%d%d%d",&a,&b,&c,&d,&e)
    sum = float64(a + b + c + d + e)
    avg = sum / 5
    fmt.Printf("%.1f",avg)
}